common/global_cpu_executor.cpp

```diff
@@ -18,17 +18,23 @@
 
 #include "common/global_cpu_executor.h"
 
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
 #include "wangle/concurrent/CPUThreadPoolExecutor.h"
 
+DEFINE_int32(cpu_thread_pool_log_evey_n, 1000, "");
+
 namespace common {
 
 wangle::CPUThreadPoolExecutor* getGlobalCPUExecutor() {
   static const auto n_threads = sysconf(_SC_NPROCESSORS_ONLN);
   static const auto queue_sz = (1 << 14);
 
-  LOG(INFO) << "Running global CPU thread pool with "
-            << n_threads << " threads, and "
-            << queue_sz << " length queue.";
+  LOG_EVERY_N(INFO, FLAGS_cpu_thread_pool_log_evey_n)
+                           << "Running global CPU thread pool with "
+                           << n_threads << " threads, and "
+                           << queue_sz << " length queue.";
```

Comment:
We could either store a pointer at the client side code, or make two anonymouse functions. One to log and return n_threads, one to log and return queue_sz. And call them below.